views/assets/styles/resume.css

```
@@ -43,6 +48,13 @@ p {
     flex-direction: row;
     justify-content: center;
     align-items: center;
+    overflow-x: auto;
+    max-width: 100%;
 
     margin: 0;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)